### ISSUE TITLE
Portion Issue

### ISSUE THREAD

**Issue by NealSavage** [Nosrick/crock-pot-mod#7]
I can't seem to go past 18 portions. Fabric, 1.18.2, mod version is 0.8.4+1.18.2. 

Only possible lead I can give is that the last thing I put in was a coconut slice from the Ecologics mod and perhaps that threw it off somehow.....?

**Comment by NealSavage** [comment]
I'll build another fire and crockpit to try to test if I don't add anything from the ecologics mod, can I get it past 18.

**Comment by NealSavage** [comment]
The second one I built got stuck at 9.

But! Breakthrough! They both got stuck upon me adding an 8th type of food variety. I think that's where you need to look for the problem at.

**Comment by NealSavage** [comment]
![image](https://user-images.githubusercontent.com/4138801/166858357-f9bbf0c9-3d63-4d0b-9235-38c99c19e234.png)

![image](https://user-images.githubusercontent.com/4138801/166858391-dc573cf1-0f39-47ea-846d-a109b717ab0f.png)



In case you want pics regarding the food types thing

**Comment by Nosrick** [comment]
Congratulations! You've discovered what I'm working on next, haha.

In my effort to make it possible to automate the pot, I gave it an
"inventory" with proper slots. I assumed eight input slots would be enough
for now, but I guess not for some people. ;)

If it bothers you, I can try to make it configurable ASAP, but otherwise
it's just going to take some time to work out the kinks.

I do very much appreciate you reporting it, though, so thank you.

**Comment by NealSavage** [comment]
Hahaha, yeaaahhh, apparently not for some people. I do enjoy my diverse farms. :P

I don't aim to add unnecessary work for you, so no config is needed. Besides, it'd slow down your work on the actual fix. I'll just try to keep track of how many types I've put in. Thanks for the offer though :) 

And, ofc! You're welcome.

**Comment by NealSavage** [comment]
Does the latest update, 0.8.5, address this? Not sure what localizations and experimental fabric stuff are

**Comment by NealSavage** [comment]
Chucked a bunch of stuff in and nope. Didn't mean this issue. lol.

**Comment by Nosrick** [comment]
> Does the latest update, 0.8.5, address this? Not sure what localizations and experimental fabric stuff are

Nah, it doesn't. I've not had much time to develop new features, as I'm caring full-time for a sick family member.
Localisation is basically translation; so a kind person did a Russian and Ukrainian localisation for everything, and I added that to the mod.
The "experimental fabric stuff" might not ever have been a problem; I started using a beta version of Fabric API by accident. I assume I never released the version that did, because I didn't get complaints about it haha. It's honestly nothing for most people to worry about!
